src/cogs/utils.py

```diff
@@ -141,7 +141,7 @@ async def verify(self, ctx):
             )
 
     @commands.command(
-        usage="[time] [am/pm] [original timezone] [timezone to convert to]")
+        usage="[time] [am/pm] [original timezone] [timezone to convert to]", help="For a list of timezones see: https://github.com/TheOtherUnknown/Malcolm-next/wiki/Commands#time")
     async def time(self, ctx, utime, apm, ozone, nzone):
         """Converts times from one timezone to another"""
```

Comment:
This doc comment is being ignored by the help command. Use it instead of `help=`.

Reply:
Fixed